pkg/disk/nodeserver.go

```diff
@@ -959,7 +945,8 @@ func (ns *nodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 				if bytes.Contains(output, []byte("it cannot be grown")) || bytes.Contains(output, []byte("could only be grown by")) {
 					deviceCapacity := getBlockDeviceCapacity(devicePath)
 					rootCapacity := getBlockDeviceCapacity(rootPath)
-					log.Infof("NodeExpandVolume: Volume %s with Device Partition %s no need to grown, with requestSize: %v, rootBlockSize: %v, partition BlockDevice size: %v", diskID, devicePath, requestGB, rootCapacity, deviceCapacity)
+					log.Infof("NodeExpandVolume: Volume %s with Device Partition %s no need to grown, with requestSize: %v, rootBlockSize: %v, partition BlockDevice size: %v",
+						diskID, devicePath, requestBytes, rootCapacity, deviceCapacity)
```

Comment:
Change the log capacity units to GB, for better readability.

Reply:
How about use hex? I used to see a lot of hex in ceph log. Once we need to read the log, we value simple and precise than readability for diagnosis purpose. And hex can be converted to different unit faster by hand. e.g.:

Decimal: 22014345216
GB: 20.502456665039062
Hex: 0x520284000
resource: 21498384Ki

Reply:
hex may be more versatile, but it's basically useless in a container scenario, and you can check that logs in kubernetes repo basically don't use hex.  keep using GiB would be fine. Accuracy issue could be ignore

Reply:
> you can check that logs in kubernetes repo basically don't use hex

Yes, but they also don't use floating point numbers. I'm searching for regex `Infof.+size`, all of them using either decimal bytes, or resource. Resource is not convenient when comparing different sizes in the log, so decimal bytes should be the way to go if we really want to follow Kubernetes.